libs/openchallenges/about/src/lib/about.component.html

```diff
@@ -1,4 +1,4 @@
-<main class="mat-typography">
+<main class="base mat-typography">
```

Comment:
We should prefix all the OC classes with `oc-` to easily identify project-specific classes. This is probably work for another ticket.